app/views/shared/_header.html.haml

```diff
@@ -1,22 +1,23 @@
 %section#top-header.row
   %h1.col-md-2
-    = link_to "MyFLiX"
-  %ul.col-md-4.clearfix
-    %li= link_to "Videos"
-    %li= link_to "My Queue"
-    %li= link_to "People"
-  %form.col-md-5.navbar-form(action=search_videos_path method="get")
-    .form-group
-      %input.form-control(type="text" placeholder="Search for videos here" name="search_term")
-    %button.btn.btn-default(type="submit") Search
-  #user_links.pull-right
-    %ul
-      %li.dropdown
-        %a(href="#" id="dlabel" role="button" data-toggle="dropdown" class="dropdown-toggle" data-target="#")
-          Welcome, Philip J. Fry
-          %b.caret
-        %ul.dropdown-menu(role="menu" aria-labelledby="dlabel")
-          %li
-            %a(href="#") Account
-            %a(href="#") Plan and Billing
-            %a(href="#") Sign Out
+    = link_to("MyFLiX", root_path)
+  - if logged_in?
+    %ul.col-md-4.clearfix
+      %li= link_to("Videos", home_path)
+      %li= link_to "My Queue"
+      %li= link_to "People"
+    %form.col-md-5.navbar-form(action=search_videos_path method="get")
```

Comment:
You should use Rails forms as they yield more benefits and are more readable than this pure HTML-like code.